instrumentation/jetty-httpclient/jetty-httpclient-9.2/javaagent/jetty-httpclient-9.2-javaagent.gradle

```
@@ -0,0 +1,27 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.eclipse.jetty"
+    module = 'jetty-client'
+    versions = "[9.2,9.4)"
```

Do all 9.4.X versions work correctly? You're using `9.+` wildcard in latest dep tests, which suggests that muzzle should work for 9.+ too
```suggestion
    versions = "[9.2,10)"
```

---

Ok - now we have `[9.2,9.+]`.   I have successful tests with Jetty Client 9.4 directly by hardcoding it in all the gradle files.  I am also having success with `-PtestLatestDeps=true`. for javaagent and library modules.